toolkit/tools/internal/logger/writerhook.go

```diff
@@ -31,11 +34,9 @@ func newWriterHook(writer io.Writer, level logrus.Level, useColors bool, toolNam
 
 	if toolName != "" {
 		formatter.CallerPrettyfier = func(frame *runtime.Frame) (function string, file string) {
-			const gray = 90
-
 			toolNameField := fmt.Sprintf("[%s]", toolName)
 			if useColors {
-				toolNameField = fmt.Sprintf("[\x1b[%dm%s\x1b[0m]", gray, toolName)
+				toolNameField = fmt.Sprintf(color.BlackString("[%s]"), toolName)
```

Comment:
Won't this blend with a black console background? If you haven't already, can you double-check before we merge. This part is to print the gray tools' names in the brackets btw. Example:
![image](https://github.com/microsoft/CBL-Mariner/assets/24633722/8fa781f9-0e92-4bb3-b12c-fd394a686271)

Reply:
Oh, yes thanks for pointing this out. I'll try it on a black background, and update color accordingly

Reply:
Changed it to hi-intensity yellow. This is how it looks on white and black background terminals
Also changed blue text to green/red depending on whether they could be treated as success/failure, as blue was difficult to see on black background
![image](https://github.com/microsoft/CBL-Mariner/assets/58672330/be3c5a0d-a4f1-46b8-b84e-862beeb98749)
![image](https://github.com/microsoft/CBL-Mariner/assets/58672330/5381edae-c3cb-4ab0-913c-345eaf96e85a)